sdk/tests/core/test_az_credential_client_secret.c

```diff
@@ -17,22 +17,34 @@
 
 #include <azure/core/_az_cfg.h>
 
+static az_span authority = AZ_SPAN_LITERAL_FROM_STR("https://login.microsoftonline.com/");
```

Comment:
```suggestion
static az_span msft_authority = AZ_SPAN_LITERAL_FROM_STR("https://login.microsoftonline.com/");
```

Reply:
It is an initial value. Later, we set it to other value.

Reply:
Why do we have a global static that is being modified by one test, and then used in another? If the order in which those tests run were flipped, wouldn't that potentially break the test?